scripts/modules.rollup.config.mjs

```diff
@@ -0,0 +1,42 @@
+import { babel } from '@rollup/plugin-babel';
+import json from '@rollup/plugin-json';
+
+export default {
+  name: 'videojsQualitySelector',
+  input: 'src/js/index.js',
+  output: [{
+    file: 'dist/videojs-quality-selector.cjs.js',
+    format: 'cjs',
+    globals: {
+      'video.js': 'videojs'
+    }
+  }, {
+    file: 'dist/videojs-quality-selector.es.js',
+    format: 'es',
+    globals: {
+      'video.js': 'videojs'
+    }
+  }],
+  external: [
+    'global',
+    'global/document',
+    'global/window',
+    'video.js'
+  ],
+  plugins: [
+    json(),
+    babel({
+      babelHelpers: 'bundled',
+      extensions: ['.js', '.jsx', '.es6', '.es', '.mjs', 'ts', 'tsx'],
```

Comment:
This list of extensions could be reduced quite a bit, as we don't use React or Typescript.